requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyOpenSSL<23.0
-pulpcore>=3.10,<3.25
+pulpcore>=3.15.0,<3.25
```

Comment:
This is what was needed to get the test running. (pulpcore dependency _must_ be specified with x.y.z because otherwise ~=3.10 will install 3.26; pulpcore 3.10 [with django 2.*] did not install properly for a broken dependency with django-storages and a click incompatibility.)
I am unsure, whether we can even bump the lower bound in a z release, but also I think it _is_ currently broken.

Reply:
The deprecation-warnings are firing (I believe) because we're now "newer than 3.10 and less than 3.16". Let's move this to >=3.16.0, if we're going to move it.

(Note: the deprecations aren't in pulp-certguard, which doesn't actually use dispatch() in its code...)

Reply:
I switched off deprecation checking on that branch, because it really is not so much help here.